src/collective/volto/formsupport/profiles/default/metadata.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <metadata>
-  <version>1300</version>
+  <version>1301</version>
   <dependencies>
-    <!--<dependency>profile-plone.app.dexterity:default</dependency>-->
+    <dependency>profile-collective.volto.otp:default</dependency>
```

Comment:
this is not related to this PR.

Reply:
https://github.com/collective/collective.volto.formsupport/pull/64#issuecomment-2263197498